vlib/v/checker/tests/par_expr_assign_void_right_type_err.out

```diff
@@ -0,0 +1,3 @@
+vlib/v/checker/tests/par_expr_assign_void_right_type_err.vv:1:3: error: assignment mismatch: 1 variable(s) but `print()` returns 0 value(s)
```

Comment:
Regenerate if you change the error message as suggested.

Reply:
Done